.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Feature Request
+about: Create a feature request for a functionality that does not currently exist in the ONNX.
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+ 
+
+---
+# Feature Request
+
+### System information
+ONNX version (you are using):
+
+### What is the problem that this feature solves?
+Please detail the discrepancy with our current functionality.
+
+### Describe the feature
+Why is this feature necessary? What does it accomplish?
```

Comment:
Also add a section for - Describe the alternatives you have considered. (A clear and concise description of any alternative solutions or features you've considered.)